Docs/user_guide/modules/sequences.md

```diff
@@ -91,7 +91,7 @@ Authors:
 
 ## Acknowledgements
 
-This work is funded in part by An Applied Cancer Research Unit of Cancer Care Ontario with funds provided by the Ministry of Health and Long-Term Care and the Ontario Consortium for Adaptive Interventions in Radiation Oncology (OCAIRO) to provide free, open-source toolset for radiotherapy and related image-guided interventions.
+This work is funded in part by an Applied Cancer Research Unit of Cancer Care Ontario with funds provided by the Ministry of Health and Long-Term Care and the Ontario Consortium for Adaptive Interventions in Radiation Oncology (OCAIRO) to provide a free, open-source toolset for radiotherapy and related image-guided interventions.
```

Comment:
```suggestion
This work is funded in part by the Applied Cancer Research Unit of Cancer Care Ontario with funds provided by the Ministry of Health and Long-Term Care and the Ontario Consortium for Adaptive Interventions in Radiation Oncology (OCAIRO) to provide a free, open-source toolset for radiotherapy and related image-guided interventions.
```

Reply:
I believe “an Applied Cancer Research Unit of Cancer Care Ontario“ is correct over “the Applied Cancer Research Unit of Cancer Care Ontario“ as there is more than one research unit of Cancer Care Ontario and the specific one is not being mentioned here.

I Google searched and found the usage of “On-PROST is an Applied Cancer Research Unit of Cancer Care Ontario, with funds provided by the Ministry of Health and Long-Term Care.”

Reply:
Thanks for looking this up. Let's ignore my suggestion then.